packages/router-generator/src/generator.ts

```diff
@@ -599,7 +602,7 @@ export async function generator(config: Config) {
 }`,
         ]),
     '// Create and export the route tree',
-    `export const routeTree = rootRoute.addChildren([${routeConfigChildrenText}])`,
+    `export const routeTree = rootRoute.addChildren({${routeConfigChildrenText}})`,
```

Comment:
should the object vs array syntax be configurable?

Reply:
I'm not sure. I like the idea that if we can confirm one way of building the route tree is more performance then we should prefer that for file based because I'm not sure if users care in this case if it is a tuple or object?

Reply:
ok, let's default to objects for filebased routing

Reply:
Yeah, I agree. Plus we've been consistent about the fact that whatever is output from the router-generator is in our control anyway.